File: Data Structures & Algorithms/remove-element/submission-0.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        i = 0
        n = len(nums)
        while i<n:
            if(nums[i]==val):
                del nums[i]
                i -=1
                n-=1
            i+=1
        return len(nums)



        
        

File: Data Structures & Algorithms/duplicate-integer/submission-0.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        unordered_map<int, int> umap;
        int n = nums.size();

        for(int i= 0 ;i<n;i++)
        {
            if(umap.find(nums[i])==umap.end()){
                umap[nums[i]]++;
                cout << umap.size() << endl;
            }
            else 
                return true; 
        }
        return false;
    }
};
